Core/DISET/private/Transports/PlainTransport.py

```diff
@@ -15,7 +15,7 @@ def initAsClient( self ):
     if 'timeout' in self.extraArgsDict:
       timeout = self.extraArgsDict[ 'timeout' ]
     try:
-      self.oSocket.create_connection( self.stServerAddress, timeout )
+      self.oSocket = socket.create_connection(self.stServerAddress, timeout)
```

Comment:
I have tried to initialize the socket in PlainTransport.py, but it doesn't seem to work. It raises errors in BaseClient.py and RPCClient.py. I am working on it.

Do we need to fix checkSanity ? I didn't really get what this function actually does. It seems like it just returns S_OK().